CMakeLists.txt

```diff
@@ -251,11 +251,14 @@ if (WIN32)
 endif()
 cmake_dependent_option(ENABLE_PROBES_WINDOWS "build OVAL probes for the Windows OVAL tests" ${IS_WIN32} "ENABLE_PROBES" OFF)
 
+option(OPENSCAP_ENABLE_SHA1 "Enable using the SHA-1 algorithm" ON)
+option(OPENSCAP_ENABLE_MD5 "Enable using the MD5 algorithm" ON)
+
```

Comment:
my only concern would be the defaults here. While SHA1 still might be considered usable, I would argue that the MD5 should not be enabled by default.

Reply:
The impact of such change is unclear for me. Currently in upstream both SHA1 and MD5 are enabled. The question is their usage in existing SCAP content. The https://github.com/ComplianceAsCode/content/ project uses file hashing only in 2 rules (etcd_unique_ca and openssl_use_strong_entropy) and in both of them they use SHA-256. But I have no idea about third-party providers of content. They might be using these old algorithms. So I rather don't change current default behavior if I don't know the impact to the users. Can anybody shed a light on this?

Reply:
I'd say that this is not a change to consider for patch version. Even maybe not for minor (because we would kick out deprecated but still legit probe, from SCAP standard PoV).

Reply:
Anyhow, this is the vanilla build flags, all distros are free to break compatibility as they wish

Reply:
Here is the reminder: #1784